### PR TITLE
OCPBUGS-43623:clarifies wording for CIDR ranges to avoid in OVN-K

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.14-4.16

Issue:
https://issues.redhat.com/browse/OCPBUGS-43623

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-ovn-kubernetes-migration-about.adoc [hunk 1]
```diff
@@ -70,7 +70,7 @@ While the OVN-Kubernetes network plugin implements many of the capabilities pres
 * Egress router pods
 --
 
-* OVN-Kubernetes, the default network provider in {product-title} 4.14 and later versions, uses the following IP address ranges internally: `100.64.0.0/16`, `169.254.169.0/29`, `100.88.0.0/16`, `fd98::/64`, `fd69::/125`, and `fd97::/64`. If your cluster uses OVN-Kubernetes, do not include any of these IP address ranges in any other CIDR definitions in your cluster or infrastructure.
+* Before migrating to OVN-Kubernetes, ensure that these IP address ranges are not in use: `100.64.0.0/16`, `169.254.169.0/29`, `100.88.0.0/16`, `fd98::/64`, `fd69::/125`, and `fd97::/64`. OVN-Kubernetes uses these ranges internally. Do not include any of these ranges in any other CIDR definitions in your cluster or infrastructure.
```
Comment (stevsmit) [inline modules/nw-ovn-kubernetes-migration-about.adoc:73]: maybe "the following" ? 

```suggestion
* Before migrating to OVN-Kubernetes, ensure that the following IP address ranges are not in use: `100.64.0.0/16`, `169.254.169.0/29`, `100.88.0.0/16`, `fd98::/64`, `fd69::/125`, and `fd97::/64`. OVN-Kubernetes uses these ranges internally. Do not include any of these ranges in any other CIDR definitions in your cluster or infrastructure.
```

**Comment by weliang1** [comment]
@JoeAldinger 
QE has tested and confirmed that OpenShift 4.16 does not support updating the internalMasqueradeSubnet (169.254.169.0/29) during migration. Please remove this setting.
SDN2OVN migration is ipv4 single stack migration, please remove `fd98::/64`, `fd69::/125`, and `fd97::/64` from the doc.

**Pull request merged**